src/ontology/cl-edit.owl

```diff
@@ -22406,8 +22407,14 @@ EquivalentClasses(obo:CL_0008031 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeV
 
 # Class: obo:CL_0008032 (rosehip neuron)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://www.nature.com/articles/s41593-018-0205-2") obo:IAO_0000115 obo:CL_0008032 "A GABAergic interneuron in human cortical layer 1 that has large rosehip-shaped axonal boutons and compact arborization.")
```

Comment:
Given the text definition and [other sources](https://www.sciencealert.com/rosehip-neuron-discovery-absent-in-mouse-models), consider using `'only in taxon' some 'Homo sapiens'`